feat: add standby readiness probe docs (#1435)
apify/apify-docs: sources/platform/actors/development/programming_interface/actor_standby.md

Please make sure to describe your Actors, their endpoints, and the schema for their
inputs and outputs in your README.

### Readiness probe

> Before Actor standby runs are ready to serve requests, Apify platform checks if the web server is ready via a readiness probe.

Reviewer: ```suggestion
Before Actor standby runs are ready to serve requests, the Apify platform checks the web server's readiness using a readiness probe.
```

> The platform sends a GET request to the path `/` with a header `x-apify-container-server-readiness-probe`. If the header is present in the request, you can perform an early return with a simple response to prevent wasting resources. Note that you have to return a response, otherwise the Actor run will never be ready and therefore won't be able to process any requests.

Reviewer: ```suggestion
The platform sends a GET request to the path `/` with a header `x-apify-container-server-readiness-probe`. If the header is present in the request, you can perform an early return with a simple response to prevent wasting resources.

:::note Return a response

You must return a response; otherwise, the Actor run will never be marked as ready and won't process requests.

:::

```
